app/controllers/api/v1/lobbies_controller.rb

```diff
@@ -24,6 +24,54 @@ def from_code
         render :show
       end
 
+      def start
+        @lobby = Lobby.find(params[:id])
+
+        if @lobby.update(status: :in_progress)
+          Pusher.trigger(@lobby.code, Lobby::LOBBY_START, {})
+          render :show
+        else
+          render 'api/v1/model_errors', locals: { errors: @lobby.errors }, status: :unprocessable_entity
+        end
+      end
+
+      def answer
+        lobby = Lobby.find(params[:id])
+        player = Player.find_by(id: params[:player_id])
+        return head :not_found if player.nil?
+
+        delete_player_answers(lobby, player)
+        create_player_answer_model(player)
+        send_answer_count_to_pusher(lobby)
+        render :answer
+      end
+
+      def delete_player_answers(lobby, player)
+        current_question = lobby.quiz.questions.find_by(order: lobby.current_question_index)
+        @answers = current_question.answers
+        current_question.answers.each do |answer|
```

Comment:
Rails supports sending arrays as values for `where` statements.

That means that instead of the `.each` here, you could just do `PlayerAnswer.where(player: player, answer: @answers).destroy_all`